lib/api/contact.js

```diff
@@ -0,0 +1,32 @@
+const Request = require('./request')
+
+class Contact extends Request {
+
+  constructor(httpClient){
+    super()
```

Comment:
Nově se posílá `httpClient` do rodiče, tedy `super(httpClient)`

Reply:
didn't check, thought that was merged already...